OD-606 [Fix] Aligning data source fields

skrupnyk opened the pull request:
@sofiiakvasnevska @inna-bieshulia

OD-606 https://weboo.atlassian.net/browse/OD-606

## Description
Fixed wrong alignment of data source fields
**Problem:** Default Bootstrap style was limiting "container" width according to iframe size.
**Solution:** Manually set minimal width for "container" class to override Bootstrap behavior.

## Screenshots/screencasts
https://storyxpress.co/video/ksrkin0rayyr4c27d

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

tonytlwu reviewed (changes requested):
Could this have worked by simply changing the class from `container` to `container-fluid`?

https://getbootstrap.com/docs/3.4/css/#:~:text=Use%20.container-fluid%20for%20a%20full%20width%20container%2C%20spanning%20the%20entire%20width%20of%20your%20viewport.

This way we are making use of the Bootstrap functionality without needing to override its behaviour.

Rejected +1

skrupnyk commented:
> Could this have worked by simply changing the class from `container` to `container-fluid`?
> 
> https://getbootstrap.com/docs/3.4/css/#:~:text=Use%20.container-fluid%20for%20a%20full%20width%20container%2C%20spanning%20the%20entire%20width%20of%20your%20viewport.
> 
> This way we are making use of the Bootstrap functionality without needing to override its behaviour.
> 
> Rejected +1

![image](https://user-images.githubusercontent.com/84472843/131662021-d046de6e-b957-4673-8673-318804b04987.png)

Changing a class was the initial idea for the fix but then we found that "container-fluid" has inner spacing which we also need to override manually. Moving a "container" class to the <body> solves the issue but we do not have access to it. We decided to treat width due to the clearance of the solution, like "we have width issues - let's fix width" instead of "we have width issues - let's fix inner spacing".

tonytlwu commented:
@skrupnyk That's because you're using `container-fluid` wrong. Bootstrap grids are meant to use the `<div class="row"></div>` container to offset the padding you're referring to. You can see this example in the link I previously provided.

<img width="438" alt="image" src="https://user-images.githubusercontent.com/290733/131708462-5c7ff9ca-6eb4-4ae9-af63-195b335cc993.png">

skrupnyk commented:
> @skrupnyk That's because you're using `container-fluid` wrong. Bootstrap grids are meant to use the `<div class="row"></div>` container to offset the padding you're referring to. You can see this example in the link I previously provided.
> 
> <img alt="image" width="438" src="https://user-images.githubusercontent.com/290733/131708462-5c7ff9ca-6eb4-4ae9-af63-195b335cc993.png">

When adding `<div class = "row"> </div>`, a horizontal scroll appears at the bottom and it is more difficult to fix it than my previous solution.

tonytlwu commented:
@skrupnyk Thanks for pointing it out. We've fixed that centrally by updating the dropdown UI.

![Image 2021-09-03 at 7 56 04 am](https://user-images.githubusercontent.com/290733/131963573-f0e1215e-9b00-4f3b-b186-490dc38fa48e.jpg)

skrupnyk commented:
> @skrupnyk Thanks for pointing it out. We've fixed that centrally by updating the dropdown UI.
> 
> ![Image 2021-09-03 at 7 56 04 am](https://user-images.githubusercontent.com/290733/131963573-f0e1215e-9b00-4f3b-b186-490dc38fa48e.jpg)

fixed

tonytlwu reviewed (approved):
@sofiiakvasnevska @inna-bieshulia This is ready for testing

Risk: Low
Affected: All widget interface with data source providers
Verify: Data source provider is correctly laid out in the interface

Squash and merge: Use PR title
